### PR TITLE
Fix footer Jinja2 bug

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Fixes a bug with the new footer changes. Causes an error in a Jinja2 environment where its testing for `params.poweredBy.partnership` but not checking if `params.poweredBy` exists first

### How to review
Footer still works as expected

### CODE REVIEW

File: src/components/footer/_macro.njk [hunk 1]
```diff
@@ -38,8 +38,6 @@
                     "altTextId": 'poweredby-alt'
                 })
             }}
-        {% else %}
```
Comment (kruncher) [inline src/components/footer/_macro.njk:41]: Why is this part being removed? it doesn't seem to be related to the `params.poweredBy.partnership` thing which is being raised.

Comment (rmccar) [reply src/components/footer/_macro.njk:41]: If you follow the logic whatever logo is being set here is being used in the `params.poweredBy.partnership` bit below. To keep the same functionality I had to in effect move this else down there

**Review by jrbarnes9 — APPROVED** [review]
LGTM. Everything seems to output the same as expected.

**Review by kruncher — APPROVED** [review]
